src/nacl/secret.py

```diff
@@ -33,6 +33,12 @@ class SecretBox(encoding.Encodable, StringFixer, object):
     send are repudiable. For non-repudiable messages, sign them after
     encryption.
 
+    Encryption is done using `XSalsa20-Poly1305`_, and there are no practical
+    limits on the number or size of messages (up to 2⁶⁴ messages, each up to 2⁶⁴
```

Comment:
I suspect using unicode like this will cause problems somewhere (sadly). Should probably just be 
```
2 \ :sup:`64`
```

Reply:
This change causes `tox -e meta` to fail with multiple `W605 invalid escape sequence '\ '` errors.

Reply:
Sigh, okay then. I guess we'll see if unicode angers other things 😄